tests/unit/test_unit_outdated.py

```diff
@@ -10,6 +11,7 @@
 
 from pip._internal.index import InstallationCandidate
 from pip._internal.utils import outdated
```

Comment:
Changing this to import the specific objects used from this module, would be a good "good first issue" IMO.

I don't have the bandwidth to file it myself right now so if someone else can get to that before me, that'd be great! ^>^

Reply:
Added #6955.